docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnInit, Pipe, PipeTransform } from '@angular/core';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { IPlayground } from '../../../../../interfaces/IPlayground';
+import { IPlaygroundProvider } from '../../../../../interfaces/IPlaygroundProvider';
+
+
+@Pipe({name: 'highlightSearch'})
+export class HighlightSearch implements PipeTransform {
+    transform(text: string, filter: string): string {
+        const highlightIndex = text.toLowerCase().indexOf(filter.toLowerCase());
+        return (highlightIndex < 0) ?
+               text :
+               text.substr(0, highlightIndex) +
+                 '<b>' + text.substr(highlightIndex, filter.length) + '</b>' +
+                 text.substr(highlightIndex + filter.length);
+    }
+}
+
+
+@Component({
+    selector: 'uxd-components-input-dropdown',
+    templateUrl: './input-dropdown.component.html',
+    styleUrls: ['./input-dropdown.component.less']
+})
+@DocumentationSectionComponent('ComponentsInputDropdownComponent')
+export class ComponentsInputDropdownComponent extends BaseDocumentationSection implements IPlaygroundProvider, OnInit {
+
+    selected: RadioOption;
+    optionList: ReadonlyArray<RadioOption> = [
+        { name: 'One' }, { name: 'Two' }, { name: 'Three' }, { name: 'Four' }
+    ];
+    filteredOptionList: ReadonlyArray<RadioOption> = this.optionList;
+    filter: string = '';
+    allowNull: boolean = false;
+    componentMaxHeight: number = 400;
```

Comment:
Perhaps rename this to just `maxHeight` for consistency as all other variables are the same as the Input property

Reply:
Done.